tests/misc_env/snmp_destination.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Tuple
+
+from paramiko.channel import ChannelFile
+
+from ceph.ceph_admin.helper import GenerateServiceSpec
+from ceph.utils import get_node_by_id
+from utility.log import Log
+
+LOG = Log(__name__)
+
+
+def exec_command(node, **kwargs: Any) -> Tuple:
```

Comment:
if this is definition needed, please add under helper module as an independent function.

Reply:
Removed this function and now addressed with node.exec_command